config/silabs/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ if (CONFIG_CHIP)
 include(ExternalProject)
 include(../zephyr/ota-image.cmake)
 include(../zephyr/zephyr-util.cmake)
-include(generate_factory_data.cmake)
+#include(generate_factory_data.cmake)
```

Comment:
do not check in commented out code or explain wit a comment why it is temporarely commented out.